docs/source/tutorials/NISAR/Nisar_access_explore.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Recommend changing " is available from NISAR mission is " to " is available from the NISAR mission that is ".</p><p>Recommend splitting into two sentences -&gt; "launched in 2014. At this time" and changing "will get from NISAR satellite." to "will get from the NISAR satellite."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>